manifests/kustomize/base/params.env

```diff
@@ -18,3 +18,10 @@ containerRuntimeExecutor=docker
 ## should be updated by defult for a versioned pipeline or not when a new
 ## version is uploaded. This sets the deployment wide definition.
 autoUpdatePipelineDefaultVersion=true
+
+
+## cronScheduleTimezone: States the timezone which should be used for
+## the cron scheduler. If not specified the local timezone of the 
+## cluster will be used. Valid values are UTC, Local or values according to 
+## the IANA Time Zone database, such as "America/New_York" and "Asia/Shanghai".
+cronScheduleTimezone='UTC'
```

Comment:
Can we add an alpha notice here when the feature is first introduced?
Regarding https://github.com/kubeflow/pipelines/blob/master/docs/release/feature-stages.md

Reply:
Sure, is this what you mean by alpha notice:
```
## Feature stage:
## [Alpha](https://github.com/kubeflow/pipelines/blob/07328e5094ac2981d3059314cc848fbb71437a76/docs/release/feature-stages.md#alpha)
 
```

and concerting when it was first introduced, what do we usually use, the pr ref?

Reply:
Yes, that looks good.
There is no standard yet.

I am thinking of creating a tracking issue for the feature to go until beta, ga. What do you think?

Reply:
> I am thinking of creating a tracking issue for the feature to go until beta, ga. What do you think?

Sounds good to me, maybe that we should create a new filter related to tracking feature readiness to make it easier to organise.